app/server/versions/v1/db/startUp.js

```diff
@@ -60,9 +60,14 @@ const mongoStartUp = async (db, collections) => {
         createdOn: timeStamp,
         latestActivity: timeStamp,
         members: [],
+        removed: false,
```

Comment:
Is there a catch in place to guard against the 'Welcome' community being deleted? I'm thinking of a situation where the first user to login to the app immediately leaves the 'Welcome' community.

Reply:
Hahah I was partially thinking about this. I have added to the startup script to add removed: false to existing communities but didn't think about the Welcome community. I'll add that, thanks!